### PR TITLE
FEAT : Add admistration user page

### PR DESCRIPTION
**Pull request by PierreA44**

### CODE REVIEW

**Review by akornne — APPROVED** [review]

**Review by Mewlyscin — APPROVED** [review]

File: backend/src/models/UserManager.js [hunk 1]
```diff
@@ -30,9 +30,12 @@ class UserManager extends AbstractManager {
   // The Rs of CRUD - Read operations
 
   async readAll() {
-    const [rows] = await this.database.query(`SELECT * FROM ${this.table}`);
+    const [rows] = await this.database.query(
+      `SELECT u.*, r.type AS role FROM ${this.table} AS u JOIN role AS r ON r.id = u.role_id`
+    );
     return rows;
   }
+  // , COUNT(rc.recipe_id) AS total_recipe JOIN recipe_comment AS rc ON rc.user_id=u.id
```
Comment (ioayoub) [inline backend/src/models/UserManager.js:38]: 👀

File: frontend/src/pages/HomePage.jsx [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import { NavLink, useLoaderData, useOutletContext } from "react-router-dom";
 import { useState } from "react";
-
-import timer from "../assets/timer/minuteur.png";
+import Carousel from "../components/carousel/Carousel";
+// import timer from "../assets/timer/minuteur.png";
```
Comment (ioayoub) [inline frontend/src/pages/HomePage.jsx:4]: 👀

File: frontend/src/pages/UserPage.jsx [hunk 3]
```diff
@@ -1,25 +1,26 @@
-import { NavLink, Outlet } from "react-router-dom";
+import { NavLink, Outlet, useOutletContext } from "react-router-dom";
 
 export default function UserPage() {
+  const { auth } = useOutletContext();
   const navUser = [
     {
       id: 1,
-      path: "/user/info",
+      path: `/user/info/${auth.id}`,
```
Comment (ioayoub) [inline frontend/src/pages/UserPage.jsx:8]: This is a security issue, you already have all needed informations to retrieve the current user in your database.

**Review by ioayoub — CHANGES_REQUESTED** [review]

**Review by QuentinVerwaerde — APPROVED** [review]

**Comment by PierreA44** [comment]
NE PAS MERGER!!!!!!

**Pull request merged**